vmware-event-router/hack/create_k8s_config.sh

```diff
@@ -48,50 +50,84 @@ fi
 echo ""
 
 echo -e "${magenta}"
-echo -n "OpenFaaS Admin Password: "
-read -s OPENFAAS_PASSWORD
-if [[ -z "${OPENFAAS_PASSWORD}" ]]; then
+read -p "Deploy OpenFaaS: [y|n] " DEPLOY_OPENFAAS
+if [[ -z "${DEPLOY_OPENFAAS}" ]]; then
     cecho "Please start over. No input entered." $red
     exit 1
 fi
+
+if [ ${DEPLOY_OPENFAAS} == "y" ]; then
+    echo -e "${magenta}"
+    echo -n "OpenFaaS Admin Password: "
+    read -s OPENFAAS_PASSWORD
+    if [[ -z "${OPENFAAS_PASSWORD}" ]]; then
+        cecho "Please start over. No input entered." $red
+        exit 1
+    fi
+fi
 echo ""
 
 cecho "Please confirm the following settings are correct:\n" $cyan
 echo -e "\tVCENTER_SERVER=${VCENTER_SERVER}"
 echo -e "\tVCENTER_USERNAME=${VCENTER_USERNAME}"
 echo -e "\tVCENTER_PASSWORD=${VCENTER_PASSWORD}"
-echo -e "\tOPENFAAS_PASSWORD=${OPENFAAS_PASSWORD}"
+echo -e "\tDEPLOY_OPENFAAS=${DEPLOY_OPENFAAS}"
+if [ ${DEPLOY_OPENFAAS} == "y" ]; then
+    echo -e "\tOPENFAAS_PASSWORD=${OPENFAAS_PASSWORD}"
+fi
 
 echo -e "${cyan}"
 read -p "Do you want to proceed with the VEBA K8s deployment [y]: " answer
 case $answer in
     [Yy]* ) cecho "Starting Deployment ..." $green;;
-    [Nn]* ) echo "Exiting ..." $red; exit;;
-    * ) echo "Exiting ..." $red; exit;;
+    [Nn]* ) cecho "Exiting ..." $red; exit;;
+    * ) cecho "Exiting ..." $red; exit;;
 esac
 
+cecho "Checking for K8s namespace creation permission ..." $green
+kubectl auth can-i create ns > /dev/null 2>&1
```

Comment:
nit: there's a `quiet` option which only returns the exit code. Otherwise, these redirects would suppress any error message, including those related to kubectl and its subcommands itself.